threem.py

```diff
@@ -209,7 +209,7 @@ def parse_genre_string(self, s):
             if not i:
                 continue
             i = i.replace("&amp;amp;", "&amp;").replace("&amp;", "&").replace("&#39;", "'")
-            genres.append(SubjectData(Subject.THREEM, i))
+            genres.append(SubjectData(Subject.THREEM, i, weight=15))
```

Comment:
We were giving 3M classifications a weight of 1. 3M classifications aren't great, but they're a lot better than random tags.